exchange/utils.go

```diff
@@ -467,12 +467,12 @@ func buildRequestExtMultiBid(adapter string, reqMultiBid []*openrtb_ext.ExtMulti
 	adapterMultiBid := make([]*openrtb_ext.ExtMultiBid, 0)
 	for _, multiBid := range reqMultiBid {
 		if multiBid.Bidder != "" {
-			if multiBid.Bidder == adapter || isBidderInExtAlternateBidderCodes(adapter, multiBid.Bidder, adapterABC) {
+			if strings.ToLower(multiBid.Bidder) == adapter || isBidderInExtAlternateBidderCodes(adapter, strings.ToLower(multiBid.Bidder), adapterABC) {
```

Comment:
If we use `openrtb_ext.NormalizeBidderName()` here, `exchange/exchangetest/multi-bids-with-extra-bids.json` fails. Whether or not the expectations in said test are correct we can leave for a future discussion.